spec/takeaway_spec.rb

```diff
@@ -0,0 +1,47 @@
+require 'takeaway'
+require 'twilio-ruby'
+
+
+describe Takeaway do
```

Comment:
Tests look good, but could add some `before do .. end` blocks to set up some of the tests to adhere to DRY principle and might be an idea to have the name of the method being tested in the `it` block description or nested `describe`/`context` blocks.